docs/contributing/reachability.md

```diff
@@ -0,0 +1,104 @@
+# Reachability
+
+### What is Reachability?
+Reachability Analysis is a security offering designed to enhance FOSSA's security analysis by providing context on vulnerable packages. It alleviates the constraints of traditional CVE assessments through the static analysis of application and dependency code, confirming the presence of vulnerable call paths. This allows pinpointing the 3-10% of exploitable vulnerabilities amid the multitude of non-exploitable ones.
+
+### Limitations
+Reachability currently supports all Maven projects dynamically analyzed by fossa-cli. The target jar of the project must exist, prior to the analysis.
+
```

Comment:
Self-review: mention, we need Endpoint to support reachability for reachability analysis to function